src/nomad_parser_vasp/parsers/xml_parser.py

```diff
@@ -128,19 +60,7 @@ def parse(
         logger: BoundLogger,
         child_archives: dict[str, EntryArchive] = None,
     ) -> None:
-        logger.info(
-            self.__class__.__repr__() + '.parse', parameter=configuration.parameter
-        )  # give feedback to Ahmed
-        archive_parser = MetainfoParser(
-            annotation_key='xml',
-            data_object=[
-                Simulation(
-                    model_system=[ModelSystem(cell=AtomicCell)],
-                    model_method=[DFT, KMesh],
-                    output=[ElectronicEigenvalues],
-                ),
-            ],
-        )
-        xml_parser = XMLParser(filepath='vasprun.xml')  # TODO apply match
-        xml_parser.convert(archive_parser)
-        archive.data = archive_parser.data_object
+        logger.info('VasprunXMLParser.parse', parameter=configuration.parameter)
```

Comment:
if this is a matching parsee you would meed to.add parsee specs

Reply:
You mean like the filename to match? I already added it to `XMLParser`